### ISSUE TITLE
Android Auto/Automotive: Support "find charging station" App Action

### ISSUE THREAD

**Issue by johan12345** [johan12345/EVMap#185]
This action could be accessed by voice command (e.g. "Hey Google, charging stations on EVMap"). It looks like this is only supported for the en-US locale so far.
https://developer.android.com/training/cars/apps/poi#charging

**Comment by MartinH-open** [comment]
I installed this app because of this feature, but I could not get it to work with Android-auto as of today.
I don't find the limitation for the en-US locale in the documentation as of today.
Perhaps it would work today - but is not implemented in the app?

**Comment by johan12345** [comment]
exactly, it still has to be implemented in the app, that's why the issue is still open 😉

**Comment by MartinH-open** [comment]
> exactly, it still has to be implemented in the app, that's why the issue is still open 😉

Ok - thanks. Is there any voice control available for this app using Android Auto?
